sapi/phpdbg/phpdbg_rinit_hook.c

```diff
@@ -56,9 +56,9 @@ static PHP_RINIT_FUNCTION(phpdbg_webhelper) /* {{{ */
 	{
 		struct sockaddr_un sock;
 		int s = socket(AF_UNIX, SOCK_STREAM, 0);
-		int len = strlen(PHPDBG_WG(path)) + sizeof(sock.sun_family);
+		size_t len = strlen(PHPDBG_WG(path)) + sizeof(sock.sun_family);
 		char buf[(1 << 8) + 1];
-		int buflen;
+		size_t buflen;
```

Comment:
This should be an `ssize_t`, as it is used for the return value of `recv()`. Otherwise the error check will fail.

Reply:
What error check?

Reply:
Oh, yes, you're right. The infinite loop is likely the cause of CI failure. Ta :)